packages/components/src/text-control/style.scss

```diff
@@ -14,6 +14,7 @@
 .components-text-control__input[type="number"] {
```

Comment:
Did we mean to add it to the number fields only? Seems like in the screenshots in the PR description we're discussing non-number text fields.

Reply:
It's hard to tell from this PR, but the style is actually applied to all fields.

https://github.com/WordPress/gutenberg/blob/c2bdeb5f2598ccfd715ddc2dc6fcaf57d6e6dce5/packages/components/src/text-control/style.scss#L1-L18

Reply:
You're right - disregard this comment.